src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -222,28 +238,73 @@ describe('Exam assessment', () => {
-            .gracePeriod(0)
-            .build();
-        courseManagementRequest.createExam(examContent).then((examResponse) => {
-            exam = examResponse.body;
-            courseManagementRequest.registerStudentForExam(exam, studentOne);
-            courseManagementRequest.addExerciseGroupForExam(exam).then((groupResponse) => {
-                exerciseGroup = groupResponse.body;
-            });
+        courseManagementRequest.deleteCourse(course.id!);
+    });
+});
+
+function startAssessing() {
+    courseAssessment.clickExerciseDashboardButton();
+    exerciseAssessment.clickHaveReadInstructionsButton();
+    exerciseAssessment.clickStartNewAssessment();
+    cy.get('#assessmentLockedCurrentUser').should('be.visible');
+}
+
+function prepareExam(end: dayjs.Dayjs, resultDate = end.add(1, 'seconds')) {
+    cy.login(admin);
+    const examContent = new CypressExamBuilder(course)
+        .visibleDate(dayjs().subtract(1, 'hour'))
+        .startDate(dayjs())
+        .endDate(end)
+        .correctionRounds(1)
+        .examStudentReviewStart(resultDate.add(10, 'seconds'))
+        .examStudentReviewEnd(resultDate.add(1, 'minute'))
+        .publishResultsDate(resultDate)
+        .gracePeriod(0)
+        .build();
+    courseManagementRequest.createExam(examContent).then((examResponse) => {
+        exam = examResponse.body;
+        courseManagementRequest.registerStudentForExam(exam, studentOne);
+        courseManagementRequest.addExerciseGroupForExam(exam).then((groupResponse) => {
+            exerciseGroup = groupResponse.body;
         });
+    });
+}
+
+function handleComplaint(reject: boolean, exerciseType: CypressExerciseType) {
+    const complaintText = 'Lorem ipsum dolor sit amet';
+    const complaintResponseText = ' consetetur sadipscing elitr';
+
+    cy.login(studentOne, '/courses/' + course.id + '/exams/' + exam.id);
+    studentAssessment.startComplaint();
+    studentAssessment.enterComplaint(complaintText);
+    studentAssessment.submitComplaint();
+    cy.get('.message').should('contain.text', 'Your complaint has been submitted');
+
+    cy.login(instructor, '/course-management/' + course.id + '/exams');
+    examManagement.openAssessmentDashboard(exam.id!, 60000);
+    courseAssessment.clickExerciseDashboardButton();
+    exerciseAssessment.clickHaveReadInstructionsButton();
+
+    exerciseAssessment.clickEvaluateComplaint();
+    exerciseAssessment.getComplaintText().should('have.value', complaintText);
+    if (reject) {
+        examAssessment.rejectComplaint(complaintResponseText, true, exerciseType);
+    } else {
+        examAssessment.acceptComplaint(complaintResponseText, true, exerciseType);
     }
-});
+    if (exerciseType == CypressExerciseType.MODELING) {
+        cy.get('.message').should('contain.text', 'Response to complaint has been submitted');
+    } else {
+        cy.get('.message').should('contain.text', 'The assessment was updated successfully.');
+    }
```

Comment:
Why is this dependent on the exercise type?

Reply:
For some reason the modeling exercise shows a different success message then the other exercises.
Modeling:
![image](https://user-images.githubusercontent.com/1368405/219957522-63d96a68-5f7e-4b57-88b5-217c8cc7aeb9.png)

Text:
![image](https://user-images.githubusercontent.com/1368405/219957533-c99c0f39-007a-4a98-a003-86673b40ca94.png)